packages/appsync-modelgen-plugin/src/scalars/supported-directives.ts

```diff
@@ -139,10 +139,12 @@ export const directives = /* GraphQL */ `
   directive @primaryKey(sortKeyFields: [String!], queryField: String) on FIELD_DEFINITION
   directive @index(name: String!, sortKeyFields: [String], queryField: String) on FIELD_DEFINITION
 
-  directive @hasOne(fields: [String!]) on FIELD_DEFINITION
-  directive @hasMany(indexName: String, fields: [String], limit: Int = 100) on FIELD_DEFINITION
-  directive @belongsTo(fields: [String]) on FIELD_DEFINITION
+  directive @hasOne(fields: [String!], references: [String!]) on FIELD_DEFINITION
+  directive @hasMany(indexName: String, fields: [String], limit: Int = 100, references: [String!]) on FIELD_DEFINITION
```

Comment:
Does ordering matter here? The `HasManyTransformer` uses a directive definition with different ordering. [[source](https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-graphql-relational-transformer/src/graphql-has-many-transformer.ts#L50)]
`directive @hasMany(indexName: String, fields: [String!], references: [String!], limit: Int = ${defaultLimit}) on FIELD_DEFINITION`

Reply:
I don't believe so. I can't find any documentation on it though. I'll update to be safe. 

It won't matter after https://github.com/aws-amplify/amplify-category-api/pull/2342.

Reply:
Directive ordering is significant only when they're actually being applied. Order of definition is not significant.